--- app/src/main/java/com/sans/finance/di/AppModule.kt
+++ app/src/main/java/com/sans/finance/di/AppModule.kt
@@ -40,7 +40,8 @@
                 AppDatabase.MIGRATION_9_10,
                 AppDatabase.MIGRATION_10_11,
                 AppDatabase.MIGRATION_11_12,
-                AppDatabase.MIGRATION_12_13
+                AppDatabase.MIGRATION_12_13,
+                AppDatabase.MIGRATION_13_14
             )
             .addCallback(callback)
             .build()
